src/main/java/oyster/commands/Command.java

```diff
@@ -0,0 +1,50 @@
+package oyster.commands;
+
+/**
+ * Command Class that holds an executable callback.
+ */
+public abstract class Command {
+    private String[] message;
```

Comment:
Plural form should be used on names representing a collection of objects.
```suggestion
    private String[] messages;
```